projects/dsp-ui/src/lib/viewer/representation/upload-form/upload-form.component.ts

```diff
@@ -0,0 +1,168 @@
+import { Component, OnInit, Output } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
+import { NotificationService } from '../../../action/services/notification.service';
+import { UploadedFileResponse, UploadFileService } from '../../services/upload-file.service';
+
+@Component({
+    selector: 'dsp-upload-form',
+    templateUrl: './upload-form.component.html',
+    styleUrls: ['./upload-form.component.scss']
+})
+export class UploadFormComponent implements OnInit {
+
+    readonly fromLabels = {
+        upload: 'Upload file',
+        drag_drop: 'Drag and drop or click to upload'
+    };
+    readonly resourceTyoe = 'Image';
```

Comment:
typo in `resourceType`; but this should come from an `@Input` parameter depending on resource class. It should be of representation format defined in knora-base ontology (<https://docs.knora.org/02-knora-ontologies/knora-base/#representations>). We can use the `Constants` from js-lib here. I can help you with that...

Reply:
Brought `@Input` back and implemented simple solution based on `resourceType as string` passed from the parent with b4143c5. Have no idea which consts are you talking about, but we can either base on instance of type/class.

Reply:
The `@Input` name should be `representation` similar to the definition from ontology and knora-api <https://docs.knora.org/02-knora-ontologies/knora-base/#representations>

Type `string` is fine.

The switch you use should go through the list of representations defined in knora-base ontology. We can access each representation definition in the `Constants` from js-lib. In this PR it would be only a case for StillImageRepresentation:
```typescript
switch (represantation) {
             case Constants.StillImageFileValue:
             ...
``` 

@tobiasschweizer is this correct? I'm not sure about the name (StillImageFileValue or HasStillImageFileValue) comes from `ResourceClass`.

Reply:
I think it depends on whether you refer to the value class or the property.

Reply:
Or if you refer to the resource class, it would be a `knora-api:StillImageRepresentation` or a subclass of this class.

An example would be an `anything:ThingPicture`: https://github.com/dasch-swiss/dsp-js-lib/blob/main/test/data/api/v2/values/get-still-image-file-value-response.json

Reply:
Yes `knora-api:StillImageRepresentation` this is what I thought first. Btw this constant does not exist yet in DSP-JS-Lib s. DSP-872

Reply:
We would have to check for subclasses too. We would use the ontology and see if a given resource class is a `knora-api:StillImageRepresentation` or a subclass.

Reply:
@kilchenmann done in 199bcc6.